collection-de-l-ecole-francaise-de-rome-full-note.csl

```diff
@@ -0,0 +1,319 @@
+      <text variable="publisher-place"/>
+    </macro>
+    <macro name="year-date">
+      <choose>
+        <if variable="issued">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="edition">
+      <choose>
+        <if is-numeric="edition">
+          <group delimiter=" ">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short" text-case="lowercase"/>
+          </group>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="thesis">
+      <group prefix=" " delimiter=", ">
+        <text macro="title" font-style="italic"/>
+        <text variable="genre"/>
+        <text variable="publisher"/>
+        <text macro="year-date"/>
+      </group>
+    </macro>
+    <macro name="articles">
+      <group delimiter=", ">
+        <text macro="title" quotes="true"/>
+        <group delimiter=" ">
+          <text term="in"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+        <text variable="volume"/>
+        <text variable="issue"/>
+        <group delimiter=" ">
+          <text macro="year-date"/>
+          <group prefix="(" suffix=")" delimiter=", ">
+            <text variable="URL"/>
+            <text macro="access"/>
+          </group>
+        </group>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <macro name="book">
+      <group delimiter=", ">
+        <text macro="title"/>
+        <text macro="edition"/>
+        <text macro="publisher"/>
+        <group delimiter=" ">
+          <text macro="year-date"/>
+          <text macro="collection"/>
+        </group>
+      </group>
+    </macro>
+    <macro name="dictionary">
+      <group delimiter=", ">
+        <text macro="title" quotes="true"/>
+        <group delimiter=" ">
+          <text term="in"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="volume" form="short"/>
+          <text variable="volume"/>
+        </group>
+        <text macro="year-date"/>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <macro name="chapter">
+      <group delimiter=" ">
+        <text macro="title" quotes="true" suffix=","/>
+        <text term="in"/>
+        <text macro="editor" suffix=","/>
+        <text variable="container-title" font-style="italic" suffix=","/>
+        <text macro="publisher" suffix=","/>
+        <group delimiter=" " suffix=",">
+          <text macro="year-date"/>
+          <text macro="collection"/>
+        </group>
+        <text macro="pages"/>
+      </group>
+    </macro>
+    <citation disambiguate-add-year-suffix="true" et-al-subsequent-min="2" et-al-subsequent-use-first="1">
```

Comment:
Indeed... I would say the whole idea of a full note with a call back to 
a previous note is odd, but modern and contemporary historians seem to 
prefer this to a sharp author-note, maybe because they mainly cite 
archives and not so much litterature.
Given the first citation gives all of the names and the reader can find 
this first citation with the indication of the note, I thought 
interesting to gain space in notes, it is more easier for reading. The 
editor doesn't seem to have specification regarding this.

Le 07/12/2015 20:19, Sebastian Karcher a écrit :
>
> In collection-de-l-ecole-francaise-de-rome-full-note.csl 
> <https://github.com/citation-style-language/styles/pull/1805#discussion_r46863309>:
>
> > +    </macro>
> > +    <macro name="chapter">
> > +      <group delimiter=" ">
> > +        <text macro="title" quotes="true" suffix=","/>
> > +        <text term="in"/>
> > +        <text macro="editor" suffix=","/>
> > +        <text variable="container-title" font-style="italic" suffix=","/>
> > +        <text macro="publisher" suffix=","/>
> > +        <group delimiter=" " suffix=",">
> > +          <text macro="year-date"/>
> > +          <text macro="collection"/>
> > +        </group>
> > +        <text macro="pages"/>
> > +      </group>
> > +    </macro>
> > +    <citation disambiguate-add-year-suffix="true" et-al-subsequent-min="2" et-al-subsequent-use-first="1">
>
> you really mean to set et-al-min at 2? That's extremely odd, no?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/citation-style-language/styles/pull/1805/files#r46863309>.
>